src/natcap/invest/ui/wind_energy.py

```diff
@@ -11,8 +11,7 @@ def __init__(self):
             label='Wind Energy',
             target=wind_energy.execute,
             validator=wind_energy.validate,
-            localdoc='wind_energy.html',
-            suffix_args_key='suffix'
```

Comment:
Ah, I see. Yea, this looks fine to remove to stick with the standardized format of InVEST Spec. I wonder if any other models have this issue.

Reply:
I briefly looked through all the other model UI files, and none of them were using `suffix_args_key` in the model `__init__()`, so I think it's okay to remove the option